install/0000_30_machine-api-operator_11_deployment.yaml

```diff
@@ -68,6 +68,10 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: metadata.namespace
+        - name: POD_NAME
```

Comment:
needed for the getting the controller ref for the recorder for the feature gate accessor...